src/components/home/profilePage/ProfilePage.jsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import "./style.scss";
+function ProfilePage() {
+  function handleClick(e) {
+    e.preventDefault();
+  }
+  return (
+    <div className="navbarpp">
```

Comment:
camelcase, and what does pp stand for, needs more clear name

Reply:
yeah sorry i fixed it